README.md

```diff
@@ -108,6 +108,73 @@ HTML styled "just right".
 Jekyll will print a local URL where the site can be viewed (usually,
 [http://0.0.0.0:4000/](http://0.0.0.0:4000/)).
 
+### Testing using Docker environment 
+
+#### Build environment
+A containerized development environment can be built using the local
+Dockerfile.
+
+Run the build-images.sh script to generate the development environment and
+associated images.
+
+```bash
+./_scripts/build-images.sh
+```
+
+This action will produce two containers: `webdev` and `webdev-validator`.
+The webdev container will execute a `jekyll serve` command with the
+polling option enabled.
+
+This provides the ability to immediately review rendered content changes.
```

Comment:
I think this would expose the rendered site at `http://127.0.0.1:4000/` ?

```suggestion
This provides the ability to immediately review rendered content changes at http://127.0.0.1:4000/ .
```

Reply:
Change has been included.